packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx

```diff
@@ -88,18 +88,19 @@ export const MarkdownPreviewStyles = createGlobalStyle<MarkdownPreviewStylesProp
     font-weight: 400;
     color: ${Colors.Black[500]};
   }
+  
+  .markdown-preview li + li {
+    margin-top: 8px;
+  }
 
   .markdown-preview ul {
-    display: grid;
-    grid-row-gap: 8px;
     width: 100%;
     margin: 0;
     padding-left: 24px;
     list-style: none;
   }
 
   .markdown-preview ol {
-    display: grid;
     grid-row-gap: 8px;
```

Comment:
Thank you! Sorry I didn't notice you had fixed it for `ul` only.

Just one small detail:
```suggestion
```